airflow/auth/managers/fab/security_manager_override.py

```diff
@@ -0,0 +1,218 @@
+        if not self.appbuilder.app.config.get("FAB_ADD_SECURITY_VIEWS", True):
+            return
+
+        if self.auth_user_registration:
+            if self.auth_type == AUTH_DB:
+                self.registeruser_view = self.registeruserdbview()
+            elif self.auth_type == AUTH_OID:
+                self.registeruser_view = self.registeruseroidview()
+            elif self.auth_type == AUTH_OAUTH:
+                self.registeruser_view = self.registeruseroauthview()
+            if self.registeruser_view:
+                self.appbuilder.add_view_no_menu(self.registeruser_view)
+
+        self.appbuilder.add_view_no_menu(self.resetpasswordview())
+        self.appbuilder.add_view_no_menu(self.resetmypasswordview())
+        self.appbuilder.add_view_no_menu(self.userinfoeditview())
+
+        if self.auth_type == AUTH_DB:
+            self.user_view = self.userdbmodelview
+            self.auth_view = self.authdbview()
+        elif self.auth_type == AUTH_LDAP:
+            self.user_view = self.userldapmodelview
+            self.auth_view = self.authldapview()
+        elif self.auth_type == AUTH_OAUTH:
+            self.user_view = self.useroauthmodelview
+            self.auth_view = self.authoauthview()
+        elif self.auth_type == AUTH_REMOTE_USER:
+            self.user_view = self.userremoteusermodelview
+            self.auth_view = self.authremoteuserview()
+        else:
+            self.user_view = self.useroidmodelview
+            self.auth_view = self.authoidview()
+
+        self.appbuilder.add_view_no_menu(self.auth_view)
+
+        # this needs to be done after the view is added, otherwise the blueprint
+        # is not initialized
+        if self.is_auth_limited:
+            self.limiter.limit(self.auth_rate_limit, methods=["POST"])(self.auth_view.blueprint)
+
+        self.user_view = self.appbuilder.add_view(
+            self.user_view,
+            "List Users",
+            icon="fa-user",
+            label=lazy_gettext("List Users"),
+            category="Security",
+            category_icon="fa-cogs",
+            category_label=lazy_gettext("Security"),
+        )
+
+        role_view = self.appbuilder.add_view(
+            self.rolemodelview,
+            "List Roles",
+            icon="fa-group",
+            label=lazy_gettext("List Roles"),
+            category="Security",
+            category_icon="fa-cogs",
+        )
+        role_view.related_views = [self.user_view.__class__]
+
+        if self.userstatschartview:
+            self.appbuilder.add_view(
+                self.userstatschartview,
+                "User's Statistics",
+                icon="fa-bar-chart-o",
+                label=lazy_gettext("User's Statistics"),
+                category="Security",
+            )
+        if self.auth_user_registration:
+            self.appbuilder.add_view(
+                self.registerusermodelview,
+                "User's Statistics",
```

Comment:
Copy/paste? This should say something about Registration

Reply:
There might be some misnaming somewhere but I reuse the code currently in main. See [here](https://github.com/apache/airflow/blob/main/airflow/www/fab_security/manager.py#L809). I cannot modify the label because that would be breaking change with current implementation.